lib/src/components/buttons/base_button.dart

```diff
@@ -67,4 +71,8 @@ abstract class DSFRBaseButton extends StatelessWidget {
   final IconPosition iconPosition;
 
   final bool iconOnly;
+
+  final MainAxisSize? mainAxisSize;
 }
+
+mixin GroupeableButton on DSFRBaseButton {}
```

Comment:
This is not really the OO way of doing things. GroupeableButton should be an interface, because it specifies what a class is, it's not an implementation

[From an interessting SO post](https://stackoverflow.com/questions/45901297/when-to-use-mixins-and-
when-to-use-interfaces-in-dart)

> Mixins is all about how a class does what it does, it's inheriting and sharing concrete implementation. Interfaces is all about what a class is, it is the abstract signature and promises that the class must satisfy. It's a type.

So I think it's better to have GroupeableButton declared as follow 

`abstract class GroupeableButton implements DSFRBaseButton {}`

Then you can use it like this 

`class DSFRPrimaryButton extends DSFRBaseButton implements GroupeableButton`

Reply:
In the end I've just made a new abstract class:
```dart
abstract class DSFRGroupeableButton extends DSFRBaseButton {
  // ...
}
```

Reply:
nice